tests/queries/0_stateless/03362_join_where_false_76670.sql

```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS t0;
+DROP TABLE IF EXISTS t1;
+
+CREATE TABLE t0(x Int) ENGINE = MergeTree ORDER BY tuple();
+
+CREATE TABLE t1(x Int) ENGINE = MergeTree ORDER BY tuple();
```

Comment:
Maybe we should populate them with data to ensure the test still tests something if we implement some shortcuts for joining empty tables?